doc/rados/configuration/mon-config-ref.rst

```diff
@@ -398,15 +398,7 @@ by setting it in the ``[mon]`` section of the configuration file.
 :Description: Position between pool's ``cache_target_full`` and
               ``target_max_object`` where we start warning
 :Type: Float
-:Default: ``0.66``
-
-
-``mon health data update interval``
-
-:Description: How often (in seconds) the monitor in quorum shares its health
-              status with its peers. (negative number disables it)
-:Type: Float
-:Default: ``60``
+:Default: ``0.660000``
```

Comment:
i don't think we need to use the default precision used by `printf()` when printing a float.

IMHO, `0.66` is more human-readable than `0.66000`.